src/OpenLoco/src/Drawing/SoftwareDrawingContext.cpp

```diff
@@ -54,6 +54,7 @@ namespace OpenLoco::Drawing
         static PaletteMap::Buffer<8> _textColours{ 0 };
         static uint16_t getStringWidth(const char* buffer);
         static std::pair<uint16_t, uint16_t> wrapString(char* buffer, uint16_t stringWidth);
+        static std::tuple<uint16_t, uint16_t, uint16_t> wrapStringTicker(char* buffer, uint16_t stringWidth, uint16_t numCharacters);
```

Comment:
I would prefer to have this return a struct. The usage of functions should be clear without having to look up what the return values are.

Reply:
Turns out it doesn't need any of the other stuff so will just make it return the one thing it needs